examples/purchaseTesterTypescript/App.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * Generated with the TypeScript template
+ * https://github.com/react-native-community/react-native-template-typescript
+ *
+ * @format
+ */
+
+ import React, { useEffect } from 'react';
+
+import { Platform, Text } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import Purchases from 'react-native-purchases';
+
+import HomeScreen from './app/screens/HomeScreen';
+import CustomerInfoScreen from './app/screens/CustomerInfoScreen';
+import OfferingDetailScreen from './app/screens/OfferingDetailScreen';
+
+import APIKeys from './app/APIKeys';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+const Stack = createNativeStackNavigator();
+
+const App = () => {
+  const isSetup = () => {
```

Comment:
this is nitpicky, but could we rename to something like `hasKeys`? just since setup is now a little overloaded... isSetup could also mean whether Purchases is configured

Reply:
This is a much better name 😊 Love it!